jobs/get_all_cards_by_date_range_gold.py

```diff
@@ -0,0 +1,89 @@
+"""
+This script will fetch all the cards for the given user according to the below criteria:
```

Comment:
Also, can you rename the script to indira_gold_table or something similar which has your name in it, so that it is identifiable?

Reply:
Renamed the script to indira_gold_table.py